### PR TITLE
Add FP64 support

### PR DESCRIPTION
**Pull request by KKyang**
Gemm only

### CODE REVIEW

File: clients/include/norm.hpp [hunk 1]
```diff
@@ -262,6 +262,8 @@ bool norm_check(double norm_error)
 {
     if(std::is_same<T, float>{})
         return norm_error < 0.00001;
+    if(std::is_same<T, double>{})
+        return norm_error < 0.00001;
```
Comment (jichangjichang) [inline clients/include/norm.hpp:266]: in Tensile client, threadhold is 0.000000000001; // 15 digits of precision - 2

**Review by jichangjichang — COMMENTED** [review]
will you put logic yaml for gfx94x?

**Comment by KKyang** [comment]
> will you put logic yaml for gfx94x?

No

File: clients/include/norm.hpp (hunk 1 above)
Comment (KKyang) [reply clients/include/norm.hpp]: Fixed

File: clients/benchmarks/client.cpp [hunk 2]
```diff
@@ -461,6 +464,11 @@ try
          value<std::string>(&filter),
          "Simple strstr filter on function name only without wildcards")
 
+        ("api_method",
+         value<int>(&api_method)->default_value(0),
+         "Use extension API. 0: C style API. 1: declaration with C hipblasLtMatmul Leyout/Desc but set, initialize, and run the problem with C++ extension API. 2: Using C++ extension API only. "
```
Comment (jichangjichang) [inline clients/benchmarks/client.cpp:469]: fix typo : leyout

Comment (KKyang) [reply clients/benchmarks/client.cpp]: Fixed

**Comment by KKyang** [comment]
> > > will you put logic yaml for gfx94x?
> > 
> > 
> > No
> 
> I didn't see you limit the fp64 CI test to gfx90a.

Fixed

**Review by jichangjichang — APPROVED** [review]
LGTM

**Pull request merged**